main.go

```diff
@@ -29,6 +35,32 @@ func main() {
 		cancel()
 	}()
 
+	envConf, err := FromEnv()
+	if err != nil {
+		logger.Log("Error getting environment variables %s", err)
+		os.Exit(1)
+	}
+
+	dd := dogstatsd.New("wattpad.ship-it.", logger)
+	go dd.SendLoop(time.Tick(time.Second), "udp", envConf.DataDogAddress())
+	hist := dd.NewTiming("worker.time", 1.0).With("worker", "worker:ecrconsumer", "queue", envConf.QueueName)
```

Comment:
`With(key1, val1, ..., keyN, valN)` produces metrics tags like `key1:val1`, ..., `keyN:valN`, so this would create a metric tag `worker:worker:ecrconsumer`